pkg/clusterbuilder/providers/capa/values/cluster_values.yaml

```diff
@@ -20,9 +20,9 @@ global:
       maxSize: 5
       minSize: 2
       rootVolumeSizeGB: 25
+      # spot instances are disabled for china because the pricing there is so random that it is hard to set any reliable values that will not cause e2e flaky
       spotInstances:
-        enabled: true
-        maxPrice: 0.2960
+        enabled: false
```

Comment:
This will disable spot for all CAPA test clusters. We only want it disabled for China, right?